src/assets/token.ts

```diff
@@ -132,7 +134,12 @@ export const createSupplySubstoreArray = async (
 			totalLSKSupply,
 		);
 	}
-	const LSKSupply = { tokenID, totalSupply: String(totalLSKSupply) };
+
+	const LSKSupply: SupplySubstoreEntry = {
+		tokenID,
+		totalSupply: String(BigInt(totalLSKSupply) + BigInt(legacyReserveAmount)),
```

Comment:
Please explicitly specify the type for `totalLSKSupply: bigint`;
```suggestion
		totalSupply: String(totalLSKSupply + legacyReserveAmount),
```